lib/gitlib.sh

```diff
@@ -201,3 +201,20 @@ gitlib::push_master () {
   logecho -n "Pushing$dryrun_flag master branch: "
   logrun -s git push$dryrun_flag origin master || return 1
 }
+
+##############################################################################
+# Ensure running with a synced repo.
+gitlib::repo_state () {
+  local output=$(git fetch --dry-run)
```

Comment:
also at least for me this is a no-op, since my origin for `kubernetes/release` is my fork. not sure how common this is for others.